package.json

```diff
@@ -3,7 +3,7 @@
   "private": true,
   "version": "4.0.0",
   "scripts": {
-    "postinstall": "npm run bootstrap",
+    "postinstall": "npm run   bootstrap",
```

Comment:
This was probably just a mistake, but it should be just a single whitespace.

Reply:
adapted